### PR TITLE
Show uptime in status area

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X] - no spaces) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

## 10

---

Feature request as of https://discourse.pi-hole.net/t/add-uptime-to-the-status-area-of-the-web-interface/1639

screenshot at 2017-09-16 16-41-17

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: scripts/pi-hole/php/header.php [hunk 1]
```diff
@@ -382,6 +387,8 @@ function pidofFTL()
                             echo "\"></i> Memory usage:&nbsp;&nbsp; N/A</a>";
                         }
                     ?>
+                    <br/>
+                    <a id="uptime" title="Note that the update is not updated automatically"><i class="fa fa-circle" style="color:#7FFF00"></i> Uptime: <?php echo $uptime; ?></a>
```
Comment (Mcat12) [inline scripts/pi-hole/php/header.php]: Note that the *uptime* ...?

**Comment by dschaper** [comment]
I'm not convinced that this is the direction we should be going.

**Review by WaLLy3K — CHANGES_REQUESTED** [review]
It looks odd with the green circle (because the circle itself won't need to change, but the uptime itself would look out of place otherwise), and isn't necessarily useful information to have up the top left.

**Review by AzureMarker — CHANGES_REQUESTED** [review]
Having the uptime visible does not bring enough benefit to outweigh the fact that it is out of the scope of a DNS server/ad blocker.

**Pull request closed without merging**